x-pack/plugins/actions/server/config.ts

```diff
@@ -50,6 +50,12 @@ export const configSchema = schema.object({
   rejectUnauthorized: schema.boolean({ defaultValue: true }),
   maxResponseContentLength: schema.byteSize({ defaultValue: '1mb' }),
   responseTimeout: schema.duration({ defaultValue: '60s' }),
+  cleanupFailedExecutionsTask: schema.object({
```

Comment:
hmm, doesn't this mean you HAVE to include this property, and `{}` would be fine since all the sub-properties are optional?  Or I guess config-schema is smart enough to realize all the sub-properties are optional, so the parent is also essentially optional?

Reply:
I did a local test. It's smart enough to fill in the default sub-properties automatically without requiring the root property defined.